src/declaration.h

```diff
@@ -534,6 +535,7 @@ class FuncDeclaration : public Declaration
                                         // scopes from having the same name
     VarDeclaration *vthis;              // 'this' parameter (member and nested)
     VarDeclaration *v_arguments;        // '_arguments' parameter
+    Ojbc_FuncDeclaration objc;
```

Comment:
@ibuclaw What do you think about the approach in [this branch](https://github.com/yebblies/dmd/compare/valistsucks)?